openshift/release/generate-release.sh

```diff
@@ -3,8 +3,14 @@
 source $(dirname $0)/resolve.sh
 
 release=$1
-
-quay_image_prefix="quay.io/openshift-knative/knative-serving-"
 output_file="openshift/release/knative-serving-${release}.yaml"
 
-resolve_resources config/ $output_file $quay_image_prefix $release
\ No newline at end of file
+if [ $release = "ci" ]; then
+    image_prefix="image-registry.openshift-image-registry.svc:5000/knative-serving/knative-serving-"
```

Comment:
Just to make sure you're aware. The images will appear in this location only after the e2e-tests-openshift.sh script will tag them from api.ci.openshift.org into the internal registry. And the images will disappear once CI for `openshift/knative-serving` finishes.